compose.yaml

```diff
@@ -20,43 +10,46 @@ services:
   devnet:
     container_name: devnet
     build:
-      context: ./rollups-node/test/devnet
+      context: ./ci/devnet
       dockerfile: Dockerfile
     ports:
       - 8545:8545
-    volumes:
-      - ./rollups-node/deployment.json:/usr/share/devnet/deployment.json
-      - ./rollups-node/anvil_state.json:/usr/share/devnet/anvil_state.json
   espresso-dev-node:
     image: ghcr.io/espressosystems/espresso-sequencer/espresso-dev-node:20241120-patch6
     environment:
       ESPRESSO_SEQUENCER_L1_PROVIDER: http://devnet:8545/
       ESPRESSO_SEQUENCER_API_PORT: 24000
       ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: 1s
       ESPRESSO_STATE_PROVER_UPDATE_INTERVAL: 1s
-      ESPRESSO_SEQUENCER_POSTGRES_HOST: db-espresso
+      ESPRESSO_SEQUENCER_POSTGRES_HOST: db
       ESPRESSO_SEQUENCER_POSTGRES_PORT: 5432
       ESPRESSO_SEQUENCER_POSTGRES_DATABASE: rollupsdb
       ESPRESSO_SEQUENCER_POSTGRES_USER: postgres
       ESPRESSO_SEQUENCER_POSTGRES_PASSWORD: password
     depends_on:
-      - db-espresso
+      - db
       - devnet
     ports:
       - 24000:24000
-  c_espresso:
-    container_name: c_espresso
+  node_espresso:
+    container_name: node_espresso
     build:
       context: .
       dockerfile: ./ci/Dockerfile
+      args:
+        - ROLLUPS_NODE_VERSION=2.0.0-alpha.4
     env_file:
       - ./ci/env.nodev2-espressolocal
     ports:
       - 10000:10000
+      - 10011:10011
+      - 10012:10012
+      - 8080:8080
```

Comment:
10012 is the port for inspects, 10011 is the one for JSON-RPC, and 8080 is the default one for Espresso Reader /nonce and /submit endpoints. I'm not sure if we can extract these from the env vars or something